include/libhashkit-1.0/hashkit.h

```diff
@@ -75,7 +79,7 @@ HASHKIT_API
 hashkit_string_st *hashkit_decrypt(hashkit_st *, const char *source, size_t source_length);
 
 HASHKIT_API
-bool hashkit_key(hashkit_st *, const char *key, const size_t key_length);
+bool hashkit_initialize_encryption(hashkit_st *kit, const char *key, const size_t key_length);
```

Comment:
We're way too far into the stability game, so changing the API for better names is not a good idea.